no-computer.md

```diff
@@ -11,41 +11,93 @@ _What need is met by your idea? Who are you helping? What is that person's
+I imagined if we were able to pick not just between two endings, which
+arrive through our choices, but rather construct our own endings using
+myriad modular building blocks?
 
 ### Domain
 _Describe the project's domain in five words._
 
+Creating chaotic stories without planning.
 
 ### Interface (syntax)
 _How might the user interact with the language? What does programming look 
 like? Why is this the right way to interact with the problem domain?_ 
 
+The user might have a deck of cards from which they can construct the initial
+state of their character(s), setting(s), and plots.  From this point on, the
+user might use a booklet or story guide which uses information about the
+items or number of characters found in a story to determine the next step,
+these choices build on one another, leading to a chaotic sequence of next
+steps highly dependent upon initial conditions.
+
+For example, one rule might read:
+"If at least one character has been injured, and has not yet healed, add
+the next tile from set #42."
 
 ### Operation (semantics)
 _What might happen when a program runs? How does a program interact with the
 user? What kinds of errors might occur, and how might they be communicated to
 the user?_
 
+The running of the program is the repeated lookup (by the user) of the next
+step in the story.  Each next step should have some relation to all of the
+the rules that can produce it so that the story, while chaotic, does not just
+read as a pseudorandom hodgepodge of events, but rather a silly and loosely
+connected thread of continuity.
+
+Errors would result on the behalf of the user following with an incorrect
+card, which they would either discover on their own, or not.  Either way
+is acceptable.
 
 ### Expressiveness
 _What should be easy to do in this language? What should be possible, but
 difficult? What should be impossible or very difficult?_
 
+It should be easy to generate a silly story based on simple inputs.
+
+It should be difficult to predict the next step in the story.
+
+It should be impossible to predict the ending of the story given the
+starting configuration.
 
 ### Related work
 _Are there any other DSLs in this domain? If not, describe how you know there
 aren't and conjecture why not. If so, describe them and provide links. How well 
 do they address the need? Are there any particularly admirable qualities of the
 language? Are there parts of the language you think could be improved?_
 
+The only one I can think of is the story books that I mentioned inspired me,
+which have one story along the top half of the pages, and a corresponding
+story along the bottom half, where at each step you decide if you want
+to flip the top page or the bottom page, and the different choices lead to
+either a final ending at the top or at the bottom.
+
+I think the language will be a bit clunky, especially as it will force users
+to perform manual lookup of the next step themselves.
```

Comment:
this is pretty similar to the book though, right? I would guess that users interested in this would be accustomed to this sort of manual effort to read their stories.